### PR TITLE
Fix more miscellaneous oddities around the class reference

### PR DESCRIPTION
**Pull request by Mickeon**
Continuation of #95774
See also https://github.com/godotengine/godot/pull/89036

This PR addressed yet another huge chunk of class reference oddities I remembered to note down for later while localizing the class reference. Some of these are clear typos, some are inconsistencies, some are just awkward.

I decided to separate this from the original because these may be more prickly.

Of particular note:
- Fix some things that were fixed in https://github.com/godotengine/godot/pull/89036 but came back in full force because I cannot be behind every contributor's back.
- **PROPERTY_USAGE_CLASS_IS_ENUM** called the property an "enum", whereas it's probably closer to an [enum type variable](https://learn.microsoft.com/en-us/cpp/c-language/c-enumeration-declarations?view=msvc-170), which is less ambiguous.
- Replace the few uses of "_acquire_", "_interrogate_", and "_augment_" with more common verbs.
- Replace leftover references to "spatial node" from 3.x.
- **NEVER** write "_The node (and any relevant children) should be [member CanvasItem.visible]_" ever again. I'm going to cry.

For the descriptions affected, not many tweaks have been made this time. Who knows when this will be merged anyway.

### CODE REVIEW

**Review by mhilbrunner — APPROVED** [review]

**Comment by Mickeon** [comment]
Updated the PR to replace these [random tabulation characters](https://github.com/godotengine/godot/pull/99020#discussion_r1836493491)

File: doc/classes/Object.xml [hunk 1]
```diff
@@ -343,7 +343,7 @@
 				    return "Welcome to Godot 4!"
 
 				func _init():
-				    print(self)       # Prints Welcome to Godot 4!"
+				    print(self)       # Prints Welcome to Godot 4!
```
Comment (akien-mga) [inline doc/classes/Object.xml:346]: From a quick look, I think for strings we tend to use quotes in the "# Prints ..." comments.
```suggestion
				    print(self)       # Prints "Welcome to Godot 4!"
```

File: doc/classes/XRCamera3D.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 		A camera node with a few overrules for AR/VR applied, such as location tracking.
 	</brief_description>
 	<description>
-		This is a helper spatial node for our camera; note that, if stereoscopic rendering is applicable (VR-HMD), most of the camera properties are ignored, as the HMD information overrides them. The only properties that can be trusted are the near and far planes.
+		This is a helper 3D node for our camera; note that, if stereoscopic rendering is applicable (VR-HMD), most of the camera properties are ignored, as the HMD information overrides them. The only properties that can be trusted are the near and far planes.
```
Comment (akien-mga) [inline doc/classes/XRCamera3D.xml:7]: While at it:
```suggestion
		This is a helper 3D node for our camera. Note that if stereoscopic rendering is applicable (VR-HMD), most of the camera properties are ignored, as the HMD information overrides them. The only properties that can be trusted are the near and far planes.
```

**Review by akien-mga — APPROVED** [review]

File: doc/classes/Object.xml (hunk 1 above)
Comment (Mickeon) [reply doc/classes/Object.xml:346]: This goes against my own rules to [prioritize accuracy for these "Prints" comments](https://github.com/godotengine/godot/pull/99081#issue-2649975342), but that's perhaps worth thinking about another time. I can do the change.

File: doc/classes/XRCamera3D.xml (hunk 2 above)
Comment (Mickeon) [reply doc/classes/XRCamera3D.xml]: There's two commas that enclose `if stereoscopic rendering is applicable (VR-HMD)` which makes sense, hence I'm keeping that.

File: doc/classes/Object.xml (hunk 1 above)
Comment (tetrapod00) [reply doc/classes/Object.xml]: To me it seems reasonable to use quotes for `print()`s that are mostly text but omit the quotes when printing other scalar or vector types directly. Especially if you are considering other special cases for vectors as in the linked PR. (Mostly commenting here so you can keep it in mind when reconsidering the guidelines later)

Comment (Mickeon) [reply doc/classes/Object.xml]: Yes, correct, it's a matter of reducing any possible ambiguity.

**Comment by Mickeon** [comment]
Rebased after minor conflict.

**Comment by Repiteo** [comment]
Thanks!

**Pull request merged**